frontend/src/components/Header.tsx

```diff
@@ -143,9 +144,6 @@ const Header = ({ logoPath, locale }: Props) => {
     setIsMobileNavExpanded(!isMobileNavExpanded);
   };
 
-  const title =
-    usePathname() === "/" ? t("title") : <Link href="/">{t("title")}</Link>;
-
```

Comment:
I put the link inline wrapping the logo. Please let me know if it needs to be a variable like it was with the text link.